mozilla/foundation.mozilla.org
# Airtable support for PNI products

Closes #6008 

**Link to sample test page**: https://foundation-s-6008-pni-t-mwlsud.herokuapp.com/cms/pages/104/edit/

**Sample Airtable sheet**: https://airtable.com/tblDv7WCDL2xNZAtn/viwxQjDZuERYodSs1?blocks=hide

**How to test:**
1. Edit any PNI product (publish or save draft, either one is fine)
2. Open the airtable link from above, and your PNI product should be in there.
3. Update the PNI product in Wagtail
4. Check it updates in Airtable with your new data. 

**Notes**:
Both General and Software products are in one large Airtable sheet. 
We need environment vars added to Production including:
- [x] AIRTABLE_API_KEY
- [x] AIRTABLE_BASE_KEY
- [x] AIRTABLE_BASE_URL
- [x] AIRTABLE_ENABLED (set tot True for Production, and False for review apps; after this PR is merged in)
